jetty-core/jetty-http3/jetty-http3-common/src/main/java/org/eclipse/jetty/http3/HTTP3StreamConnection.java

```diff
@@ -145,7 +160,7 @@ private void processDataFrames(boolean setFillInterest)
         }
         catch (Throwable x)
         {
-            tryReleaseInputBuffer(true);
+            tryReleaseInputBuffer(remotelyClosed);
```

Comment:
Why was this changed? In case of exceptions we want to release forcibly, since we are not going to read ever again.